01 - Community/People/Josh Plunkett

```diff
@@ -0,0 +1,65 @@
+---
+aliases:
+- Joshua Plunkett
+- Sklore
+- Sklex
+tags: 
+- 
+publish: true
+---
+
+# Josh Plunkett
+
+%% - GitHub: [agathauy](https://github.com/agathauy/) ^github %%
+
+- Discord: `@sklore#8618` ^discord
+
+
+%% - Website: <https://github.com/agathauy> ^website %%
+
+<!-- - [[Publish sites|Publish site]]: ^publish -->
+
+%% Feel free to add a bio below this comment %%
+
+## Author of
+Tutorials for using Obsidian as a TTRPG Campaign Manager. 
+- [Patreon](https://www.patreon.com/JPlunkett)
+- [YouTube](https://youtube.com/playlist?list=PLV5XWfKkFpk7MJTKv5YdSSpT9b-vLslWu)
+
+## Community
+- [Obsidian TTRPG Facebook Users Group](https://www.facebook.com/groups/obsidianttrpgusers)
+
+%% Begin Hub: Released contributions %%
+%% ### Plugins %%
+%% - [[wikilinks-to-mdlinks-obsidian|Wikilinks to MDLinks]] %%
```

Comment:
I'm confused by this line. It's commented out, but suggests that Josh Plunkett is the author of [wikilinks-to-mdlinks-obsidian](https://publish.obsidian.md/hub/02+-+Community+Expansions/02.05+All+Community+Expansions/Plugins/wikilinks-to-mdlinks-obsidian)

But that plugin's page says the author is [agathauy](https://publish.obsidian.md/hub/01+-+Community/People/agathauy)

Could you clarify please?

Reply:
I just copied an existing note and left the comments in. I'll clean that up.

Reply:
Done